html/semantics/the-button-element/command-and-commandfor/invalid-element-types.html

```diff
@@ -29,7 +29,7 @@
   const firstHalf = half === 'first';
   assert_true(firstHalf || half === 'second');
 
-  const allowed = ['button','input','menuitem'];
+  const allowed = ['button','menuitem'];
```

Comment:
Technically menuitem should probably not be included or the test should be tentative, but given it's testing a lack of support on *other* elements I think it's probably fine.

But input is not allowed, we excluded input buttons because they're old and bad.